### PR TITLE
[opensvc] Add OpenSVC plugin

### PR DESCRIPTION
**Pull request by arnaudveron**
* configuration collected from /etc/opensvc
* logs collected from /var/log/opensvc
* states collected from /var/lib/opensvc and cli commands
* as secrets are encrypted with cluster secret, obfuscation is needed
* all strings matching "secret =" are obfuscated, as per example

    Example
    -------

    root@xps13# cat ${SOSREPORT}/etc/opensvc/cluster.conf
    [DEFAULT]
    id = d3387ce0-e766-496b-8438-b0dd23e083fd

    [cluster]
    nodes = xps13
    id = 5c736690-60ed-11e9-81d7-8b360f06a5a4
    name = xps13
    secret = ****************************
    ; secret = ****************************
    vip@titi = titi

    [array#freenas.opensvc.com]
    type = freenas
    api = https://freenas.opensvc.com/api/v1.0
    username = root
    password = system/sec/freenas.opensvc.com

    [hb#2]
    secret = ****************************
    type = relay
    timeout = 30
    relay = relay.opensvc.com

Signed-off-by: Arnaud Veron <arnaud.veron@opensvc.com>

---
Please place an 'X' inside each '[]' to confirm you adhere to our [Contributor Guidelines](https://github.com/sosreport/sos/wiki/Contribution-Guidelines)

- [X] Is the commit message split over multiple lines and hard-wrapped at 72 characters?
- [X] Is the subject and message clear and concise?
- [X] Does the subject start with **[plugin_name]** if submitting a plugin patch or a **[section_name]** if part of the core sosreport code?
- [X] Does the commit contain a **Signed-off-by: First Lastname <email@example.com>**?
- [X] If this commit closes an existing issue, is the line `Closes: #ISSUENUMBER` included in an independent line?
- [X] If this commit resolves an existing pull request, is the line `Resolves: #PRNUMBER` included in an independent line?

### CODE REVIEW

**Comment by arnaudveron** [comment]
Please find attached a sosreport with opensvc plugin enabled.
[sosreport-xps13-2021-03-09-jnybjns.tar.xz.gz](https://github.com/sosreport/sos/files/6109079/sosreport-xps13-2021-03-09-jnybjns.tar.xz.gz)

File: sos/report/plugins/opensvc.py
Comment (TurboTurtle) [inline sos/report/plugins/opensvc.py:47]: Nitpick: over-indented for sos-style. List items should be indented one level from the parent line, like you have in the following `add_cmd_output`

File: sos/report/plugins/opensvc.py
Comment (TurboTurtle) [inline sos/report/plugins/opensvc.py:65]: Similar nitpick: over-indented

File: sos/report/plugins/opensvc.py
Comment (TurboTurtle) [inline sos/report/plugins/opensvc.py:26]: `add_cmd_output()` takes a single item as well as a list, so we can just pass the bare string here for each line iteration

**Review by TurboTurtle — COMMENTED** [review]
The plugin looks good to me, just a few very minor nitpicks noted below.

Great first commit to the project!

File: sos/report/plugins/opensvc.py
Comment (arnaudveron) [reply sos/report/plugins/opensvc.py]: Fixed in https://github.com/sosreport/sos/pull/2441/commits/4c7d1b1665ef2ba81690411a6ea1014abfce92b9

File: sos/report/plugins/opensvc.py
Comment (arnaudveron) [reply sos/report/plugins/opensvc.py]: Fixed in commit https://github.com/sosreport/sos/pull/2441/commits/4c7d1b1665ef2ba81690411a6ea1014abfce92b9

File: sos/report/plugins/opensvc.py
Comment (arnaudveron) [reply sos/report/plugins/opensvc.py]: Fixed in commit https://github.com/sosreport/sos/pull/2441/commits/4c7d1b1665ef2ba81690411a6ea1014abfce92b9

**Review by TurboTurtle — APPROVED** [review]
Looks good to me. Only note is that for this project, we like to squash fixup commits into their parents. You can do this via:

1. `git rebase -i HEAD ~2` from your local branch
2. Change 'pick' to 'fixup' (or just 'f') in the editor for the second commit and save
3. Force push your branch

Or I can take care of it when I merge this after a second maintainer gives their ack, either way.

**Review by pmoravec — APPROVED** [review]

**Pull request closed without merging**
